### ISSUE TITLE
Warning for space before units

### ISSUE THREAD

**Issue by g0t00** [vhdl-linter/vhdl-linter#98]
```vhdl
  constant T_PD          : time      := 2ns;
```
This is not strictly legal with the VHDL standard (should be `2 ns`). It should be warned
